### ISSUE TITLE
Some relatively large model cannot be exported as CS FMU from OpenModelica

### ISSUE THREAD

**Issue by jph-tavella** [OpenModelica/OpenModelica#8634]
### Description

I give you here the 4 parts of a large electrical model we would like to export as 4 FMUs to cosimulate them in OMSimulator.
The 4 parts are named MaintNtwk,  MVBB1, MVBB2 & MVBB3.
On my computer I can only export MaintNtwk & MVBB3 (small components) as FMUs but I cannot export MVBB1 & MVBB2 (larger models).

Perhaps a question of configuration somewhere in the tool?

### Steps to Reproduce

Easily trying to export the models from the zipped .mo files.
[temp.zip](https://github.com/OpenModelica/OpenModelica/files/8180332/temp.zip)

### Expected Behavior

Correct & runnable FMUs.

### Version and OS

 OpenModelica Version: v1.19.0-dev-637-g5d2381ceaf (64-bit)
 OS: Windows 10, 64 bit

**Comment by casella** [comment]
@jph-tavella do you get any warning or error message while trying to do so?

**Comment by jph-tavella** [comment]
@casella: here is a copy of the All tab after an attempt to export MVVB2 as an FMU

[OMlog.docx](https://github.com/OpenModelica/OpenModelica/files/8187779/OMlog.docx)
.

**Comment by casella** [comment]
```
[1] 18:20:00 Ecriture Notification
unmatched equations: 8443, 8444 
 
Index Reduction neccessary! 
```
@kabdelhak, looks like you could have a look?

@jph-tavella can this FMU export be done with Dymola?

**Comment by jph-tavella** [comment]
@casella: yes sure the 4 models composing this case (MaintNtwk, MVBB1, MVBB2 & MVBB3) can be exported in a relatively short time, about a few seconds.
And then after that the cosimulation of the 4 Dymola FMUs is done for a power flow calculation (stopTime=0) in 230 ms without Jacobian calculation and in 1.22 s with Jacobian calculation.

**Comment by jph-tavella** [comment]
@casella: I think this is a regression as with previous OM release, it was working with these flags & options:
annotation (experiment(
      StopTime=0,
      Tolerance=1e-06),
    __OpenModelica_simulationFlags(ls = "klu", lv = "LOG_STATS", nls = "kinsol", s = "dassl"),
    __OpenModelica_commandLineOptions = "--matchingAlgorithm=PFPlusExt --indexReductionMethod=dynamicStateSelection -d=initialization,NLSanalyticJacobian --TearingMethod=minimalTearing");

The issue is that these flags/options are not operating now. Even if they were operating, I think it is too ugly for users to have these kind of flags/options to set. Maybe some are not useful, and also the required ones should be automatically set if required by the tool.

**Comment by casella** [comment]
@jph-tavella,the flag  `--TearingMethod` is wrong, it should be `--tearingMethod`. Could that be the case of the regression?

**Comment by jph-tavella** [comment]
@casella: strange the flag changed its name with new release... Nevertheless I tested it and the result is the same, export fails as shown by the screenshot here:

![Capture](https://user-images.githubusercontent.com/32952030/157404281-b01fb197-08cb-4a15-9e45-3060bee01c05.PNG)

**Comment by jph-tavella** [comment]
Can you reproduce my issue or not?
